Code/Framework/AzFramework/Platform/Linux/AzFramework/API/ApplicationAPI_Linux.h

```diff
@@ -25,4 +29,22 @@ namespace AzFramework
 
         using Bus = AZ::EBus<LinuxLifecycleEvents>;
     };
+
+#if AZ_LINUX_WINDOW_MANAGER_XCB
+    class LinuxXCBConnectionManager
```

Comment:
As part of the proposed C++ Coding Standard, abbreviations are treated as regular words.

So the name of the class should be `LinuxXcbConnectionManger` with a lowercase `cb`

Reply:
Changed